modules/operator-deploy-cli.adoc

```diff
@@ -50,6 +50,60 @@ metadata:
 spec:
   configBundleSecret: init-config-bundle-secret
 ----
+
+.. If you have a proxy configured, you can add the information using overrides for Quay, Clair, and mirroring:
++
+.quayregistry.yaml:
+[source,yaml]
+----
+  kind: QuayRegistry
+  metadata:
+    name: quay37
+  spec:
+    configBundleSecret: config-bundle-secret
+    components:
+      - kind: objectstorage
+        managed: false
+      - kind: route
+        managed: true
+      - kind: mirror
+        managed: true
+        overrides:
+          env:
+            - name: DEBUGLOG
+              value: "true"
+            - name: HTTP_PROXY
+              value: quayproxy.qe.devcluster.openshift.com:3128
+            - name: HTTPS_PROXY
+              value: quayproxy.qe.devcluster.openshift.com:3128
+            - name: NO_PROXY
+              value: svc.cluster.local,localhost,quay370.apps.quayperf370.perfscale.devcluster.openshift.com
```

Comment:
here "quay370.apps.quayperf370.perfscale.devcluster.openshift.com" is the quay route name